tests/e2e/e2e_feegrant_test.go

```diff
@@ -0,0 +1,60 @@
+package e2e
+
+import "fmt"
+
```

Comment:
This is straightforward, but for completeness, would like same as test overview as in: https://github.com/cosmos/gaia/pull/1770/files#top